cmd/compute-domain-kubelet-plugin/checkpoint.go

```diff
@@ -12,14 +13,22 @@ type Checkpoint struct {
 }
 
 type CheckpointV1 struct {
-	PreparedClaims PreparedClaims `json:"preparedClaims,omitempty"`
+	PreparedClaimsByUID `json:"preparedClaimsByUID,omitempty"`
```

Comment:
oh, I forgot a `PreparedClaimsByUID` type/name definition here (depending on how you look at this). But this compiles and works. Why?

Reply:
It seems I accidentally created an embedded field here.

> A field declared with a type but no explicit field name is called an embedded field

> The unqualified type name acts as the field name.

https://go.dev/ref/spec#Struct_types

This works, but we don't want this here.